### PR TITLE
[luci-pass-value-py-test] Introduce pytest

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit introduces a pytest for luci-pass-value-test.

Related: #12071
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
### Before

```bash
Start 75: luci_pass_value_test
75/76 Test #75: luci_pass_value_test ...................   Passed   63.73 sec
```

### After

```bash
$ ./nncc test -R luci_pass_value_py_test                    
Test project /home/seongwoo/ONE/build
    Start 70: luci_pass_value_py_test
1/1 Test #70: luci_pass_value_py_test ..........   Passed    1.98 sec

100% tests passed, 0 tests failed out of 1

Total Test time (real) =   1.98 sec
```

File: compiler/luci-pass-value-py-test/conftest.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    tflite_dir = metafunc.config.getoption('tflite_dir')
+    circle_dir = metafunc.config.getoption('circle_dir')
+    eval_driver_path = metafunc.config.getoption('luci_eval_driver')
+    if list_path is None:
+        tests_default_tol = []
+    else:
+        with open(list_path) as f:
+            contents = [line.rstrip() for line in f]
+
+        comment_removed = [line for line in contents if not line.startswith('#')]
```
Comment (seanshpark) [inline compiler/luci-pass-value-py-test/conftest.py:31]: Q) is it OK to if there is `#` at the end like this?
```
addeval(model, pass) # just comment
```

File: compiler/luci-pass-value-py-test/requires.cmake [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require("common-artifacts")
+require("luci-eval-driver")
```
Comment (jinevening) [inline compiler/luci-pass-value-py-test/requires.cmake:2]:
```suggestion
require("luci-eval-driver")
require("circle2circle")
```

File: compiler/luci-pass-value-py-test/conftest.py (hunk 1 above)
Comment (mhs4670go) [reply compiler/luci-pass-value-py-test/conftest.py:31]: @seanshpark Yes. `re` module match just exact regex for this.
```python
p = re.compile('eval\((.*)\)')
```

**Pull request merged**